### PR TITLE
add momokai aurora

### PR DESCRIPTION
**Pull request by peepeetee**
## Description
Momokai aurora, a 6 key keypad with one encoder

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/20419

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/momokai/momokai_aurora.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
+{
+    "name": "Momokai Aurora",
+    "vendorId": "0x69F9",
+    "productId": "0x0009",
+    "keycodes": ["qmk_lighting"],
+    "menus": [
```
[the remark this answers is not in the document]
Comment (peepeetee) [reply v3/momokai/momokai_aurora.json:6]: I have the impression that having less than all of the animations requires the corresponding list in VIA

[the remark this answers is not in the document]
Comment (Xelus22) [reply v3/momokai/momokai_aurora.json:6]: Yes, but I think from counting you have all of them here? Unless im counting wrong.

[the remark this answers is not in the document]
Comment (peepeetee) [reply v3/momokai/momokai_aurora.json:6]: oooooooooo I must have missed it, will look into it. thanks!!!!!!!

[the remark this answers is not in the document]
Comment (peepeetee) [reply v3/momokai/momokai_aurora.json]: Changes applied. Thanks!!!!!!!!! I'll have it go in first before curating the list of animations

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**